### ISSUE TITLE
Secret Potion can be locked behind a ledge, making it impossible to obtain

### ISSUE THREAD

**Issue by TheEnhancedExe** [adrienntindall/hgss-map-randomizer#39]
I was trying for hours to find the house with the secret potion until I had eventually checked every single warp available to me with 5 badges. So I looked up its connection in the txt file and the pharmacy was connected to the Victory Road 3F ladder down NW, which is locked behind a ledge. I tried that warp in victory road ingame by using walk through walls to get to the ladder and it was the one containing the pharmacy. 

So basically, there's currently a chance that its impossible to fight Jasmine, because the Pharmacy is linked to a warp that you're unable to use. There could also be the opposite possibility that this specific warp in victory road is categorized wrong and any important warp, even a gym, could potentially get locked behind that ledge, which would make you unable to finish all gyms in order.

**Comment by adrienntindall** [comment]
can you send the logs + randomizer version please

**Comment by TheEnhancedExe** [comment]
I used V1.3.2. 

Here's the log file:
[4788 Pokemon - SoulSilver_map_randomized.log](https://github.com/adrienntindall/hgss-map-randomizer/files/8987051/4788.Pokemon.-.SoulSilver_map_randomized.log)

**Comment by adrienntindall** [comment]
Found the issue, looks like the pharmacy was mislabeled as a "garbage dead-end" despite being in the main progression, which makes the logic think it's eligible to be swapped without issue for something required but inaccessible. 
So basically something else got stuck behind that ledge and the pharmacy got swapped in there in its place, and if I were to guess it would've been one of the first four gyms since those get checked after the pharmacy does. 
The change has been pushed, so replacing the HGSS_Blocks_Shared folder you have with the HGSS_Blocks_Shared folder in the code tab will let you continue to play without issue while I work on the other things I wanted to do for the next update
